eclipse-collections-forkjoin/src/test/java/org/eclipse/collections/impl/forkjoin/FJIterateTest.java

```diff
@@ -265,7 +267,7 @@ public void testForEachImmutable()
     @Test
     public void testForEachWithException()
     {
-        Verify.assertThrows(RuntimeException.class, () -> FJIterate.forEach(
+        assertThrows(RuntimeException.class, () -> FJIterate.forEach(
```

Comment:
Suggestion to make this `Assert.assertThrows` instead of a static import, so it is consistent with the rest of the assertions in this class. Same comment for lines 331 and 495.

Reply:
We have other such instances where we us static import. I dont understand the consistency aspect here. That said, I have fixed it.